inexor/fpsgame/client.cpp

```diff
@@ -179,6 +179,7 @@ namespace game
 
     /// change own nick name
     /// @param name my new nickname
+	/// @param tag is clantag of the player
```

Comment:
I have just checked using Visual Studio and the indentation is correct, I have also used spaces to get it aligned as well. I think this is because of github putting that + sign which "shows" the comment being out of place.

![image](https://user-images.githubusercontent.com/25210476/29069640-a3c6069c-7c33-11e7-9329-d814641713ea.png)